pkg/agent/datamodel/osimageconfig.go

```diff
@@ -20,9 +20,9 @@ var (
 
 	Ubuntu1804Gen2OSImageConfig = AzureOSImageConfig{
 		ImageOffer:     "aks",
-		ImageSku:       "aks-ubuntu-1804-gen2-2021-q3",
+		ImageSku:       "aks-ubuntu-1804-gen2-2022-q1",
```

Comment:
oh, this was a good catch @chengliangli0918. I think @ganeshkumar5699 made the version PR before creating a new plan for the 2 PIR images. so there would have been a mismatch, but good catch.

Reply:
Thanks for this PR @chengliangli0918! Great catch!